hydromt_wflow/wflow.py

```diff
@@ -2715,6 +2716,267 @@ def setup_precip_forcing(
             precip_out.attrs.update({"precip_clim_fn": precip_clim_fn})
         self.set_forcing(precip_out.where(mask), name="precip")
 
+    def setup_precip_from_point_timeseries(
+        self,
+        precip_fn: Union[str, pd.Series, pd.DataFrame, xr.Dataset],
+        interp_type: str = "nearest",
+        precip_stations_fn: Union[str, gpd.GeoDataFrame] = None,
+        index_col: str = None,
+        buffer: int = 100,
+        metpy_kwargs: dict = None,
+        **kwargs,
```

Comment:
kwargs are actually not so relevant for the hydromt ``precip`` function so we can simplify into one kwargs.